src/utilities/customNumberFormat.ts

```diff
@@ -0,0 +1,27 @@
+
+    /**
+     *  Replace '.'  with ','  
+     *  Add space between thousands.
+     * @param value the value to be formatted. 
+     */
+   export function numberFormat(value:number){
+        return new Intl.NumberFormat("nb-NO").format(value);
+    }
+
+    /**
+     *  Replace '.'  with ','  
+     *  Add space between thousands.
+     * @param value the value to be formatted. 
+     * @param decimals exact number of decimals to be presented
+     */
+   export function numberFormatFraction(value:number, decimals:number){
+        return new Intl.NumberFormat("nb-NO", {minimumFractionDigits: decimals, maximumFractionDigits: decimals}).format(value);
+    }
+
+    /**
+     * 
+     * Replace  '.' with ','
+     */
+    export function replaceComma(value:string){
```

Comment:
Is this method made surperflous by the ones above it?

Reply:
In the remainderQuotients table I want to replace the comma but not separate the thousands.

Reply:
How come? Feel there is not enough space?
Maybe it's worth wrapping this function in a second function that converts decimal numbers to strings with a specified number of decimal places, and then calls this function to replace the comma? That way we don't have to do that manually each time we wish to use this function.

Reply:
A suggestion

```
    export function numberFormatDecimals(value:number, decimals: number){
        return value.toFixed(decimals).replace(".", ",");
    }
```

Reply:
The new function looks good to me.
What is the rationale behind not separating thousands for remainder quotients?

Reply:
I thought it looked better. I see that we should go for consistency so I removed the replaceComma function in the latest commit.